### DISCUSSION TITLE
Issue with Git + React + Terminal

### DISCUSSION THREAD

**Discussion by axzr** [codesandbox/codesandbox-client#6016] [General]
Hi,

I'm working on a React app and I need access to the terminal too. After some digging I found a react-node template. That worked fine until I saved my app to GitHub. I created a new repo for this app and in the process of creating the repo and saving the first version of the app my terminal "went away".

I no longer have a terminal and not sure how to get it back without starting from scratch!

Cheers.

**Comment by lbogdan** [comment]
Hey @axzr ,

Can you please link to the exact `react-node` template you used, and also to the GitHub repository you created? Thanks!

**Reply by axzr** [reply]
Hi, this was a few weeks ago, I don't have the details to hand any more. When I get the chance I'll see if I can reproduce the issue.
